src/App.js

```diff
@@ -1,13 +1,21 @@
-import React from 'react';
-import { Route, Switch } from 'react-router-dom';
+import React, { useState } from 'react';
+import { Route, Switch, Redirect } from 'react-router-dom';
 import './App.css';
-import Test from './components/Test';
 import Navigation from './components/Navigation';
 import AddItem from './components/AddItem';
 import List from './components/List';
 import Welcome from './components/Welcome';
+import getToken from './lib/tokens';
 
 function App() {
+  const [token, setToken] = useState(localStorage.getItem('userToken') || '');
```

Comment:
the conditional statement sets the `token` default value to our token from `localStorage` if it exists or to an empty string if no token has been generated yet.